backend/configs/main.conf

```diff
@@ -1,8 +1,9 @@
 ipadress=127001
-camera1=http://tuincam.bt.tudelft.nl/mjpg/video.mjpg
-camera2=http://131.180.123.51/zm/cgi-bin/nph-zms?mode=jpeg&monitor=2&scale=100&buffer=100
-camera3=http://131.180.123.51/zm/cgi-bin/nph-zms?mode=jpeg&monitor=1&scale=100&buffer=100
-camera2IP=123111
+
+camera_1=simplecamera, http://tuincam.bt.tudelft.nl/mjpg/video.mjpg
```

Comment:
Maybe it is cleaner to make this into camera_1_name and camera_1_adress and maybe even a camera_1_type.